### DISCUSSION TITLE
Aggregate a grand-parent sum of parent fields (calculated using the platforms rollup feature) based on a child field change?

### DISCUSSION THREAD

**Discussion by KeithClarke** [jamessimone/apex-rollup#361] [Q&A]
I have a situation where there are 1,000 child records on each of 10 parent objects that I need to rollup into one grandparent object.

While using apex-rollup can AFAIK handle all of this, I'm wondering if this hybrid is possible? First use a platform rollup field for the 1,000 => 1 child to parent part. Then can apex-rollup be configured to be invoked on a child record change, but skip the parent object calculation (as the platform will calculate that), and only do the grandparent rollup? This would potentially reduce the CPU and DML cost?

**Comment by jamessimone** [comment]
You can roll up directly from the grandchildren to the grandparent with apex rollup. I’ve heard of people using platform rollup fields before as part of their apex rollups, but for this situation you may find the grandparent rollups more direct. There’s a good wiki page for setting those up if you haven’t seen that already.

**Reply by KeithClarke** [reply]
Thanks! I'll give it a go using https://github.com/jamessimone/apex-rollup/wiki/Configuring-Grandparent-(or-Greater)-Rollups as my guide.
